### DISCUSSION TITLE
Thumbnail for shared Video Article?

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#578] [Q&A] [answered]
Hello Astroid Community

I am testing sharing Blog Posts on Facebook

eg 1:
https://astroidframe.work/blog/astroid-framework-3-0-10-new-widgets-added-and-tons-of-improvements-and-bug-fixes

<img width="540" alt="Astroid-Share-EG01" src="https://github.com/templaza/astroid-framework/assets/9163424/02e23e02-5f12-4c93-808c-e16539f1cf35">

eg 2:
https://astroidframe.work/blog/astroid-hands-on/how-to-get-up-and-running-the-traditional-way

<img width="540" alt="Astroid-Share-EG02" src="https://github.com/templaza/astroid-framework/assets/9163424/663b7442-5a62-424f-a7e5-d0a90456a71e">

Where:
eg 1: is a Regular Article with I guess a regular Thumbnail.
eg 2: is a Video Article with a thumbnail being pulled from You Tube

I am wondering if this is just me (possible user error?) or is it possible to get a slicker implementation for sharing a Video Article on Facebook and similar? As in having a thumbnail also for Video Articles?

thanks for any tips/help

Paul

ps if this is one for the ideas category please fell free to move it :-)

**Comment by sonvnn** [comment]
Thank you @paurray! I will research about it and response you asap.

**Comment by paurray** [comment]
I did some testing on the same article.
Using this tool here:

https://developers.facebook.com/tools/debug/

I tried 2 links:

https://finalbug.tv/de/gestaltung-planung-web-tutorials/astroid-getting-up-running-the-traditional-way
https://astroidframe.work/blog/astroid-hands-on/how-to-get-up-and-running-the-traditional-way

Also the results can be seen here:

You can see the result here:
https://www.facebook.com/groups/astroid

In the Astroid Article/Blog section on my own site I can switch ON “Enable Open Graph for your Posted Articles”
(I ?suspect? that this part is mission critical)

and also on my site in the Open Graph section of the Blog Post “Getting Up & Running the Traditional Way”
I add Title “Astroid Getting Up & Running the Traditional Way”
OG Description “The Astroid Framework completely is free and integrates seamlessly with Joomla. The Astroid install is in and of itself is pretty straight forward but there are a couple of things to look out for before making the jump. In this article we cover the Traditional Way.”
Image: Where the Image is a thumbnail that I upload here in the Open Graph Section

@sonvnn  if you get a chance could you kindly check the Astroid Article/Blog Open Graph Settings in the Super Admin Area of the Astroid Site?

<img width="960" alt="OpenGraphSettings" src="https://github.com/templaza/astroid-framework/assets/9163424/8e767de4-1795-4261-b46d-28e6c3627e5b">

**Reply by sonvnn** [reply]
Thank you! I will check it and response you asap

**Comment by Chacapamac** [comment]
@paurray, @sonvnn , I find this for using Video in Open Graph
— https://docs.oracle.com/cloud/social/doc.1/srm/vitrue-publisher/adding-a-video-to-a-post-with-open-graph-protocol.html
— https://searchengineland.com/use-open-graph-to-get-video-thumbnails-into-your-search-results-128264

**Comment by paurray** [comment]
@Chacapamac @sonvnn 

What puzzles me is that sharing from my site also built with Astroid on Facebook works. But sharing from the official Astroid Site on Facebook does not work.

My guess is that the installs are pretty similar. eg at my end Joomla 5.03 and Astroid 3.0.11! But there is something (maybe) not set correctly in the Super Admin section of the official (Not some thing that I personally want access to) Astroid site???

Above please see how I have things set above in my Astroid admin area.

Below in the Joomla/Astroid back end articles section on the Astroid Site and my Site!

<img width="960" alt="PublishedOnAstroidSITE" src="https://github.com/templaza/astroid-framework/assets/9163424/aedeb7b6-8979-472b-a93b-f619975c2857">

(above) Shared from Astroid Site on FB:  Does not Work.

<img width="960" alt="PublishedOnMyFinalBugTvSITE" src="https://github.com/templaza/astroid-framework/assets/9163424/d1d0633b-9354-46ca-b45e-ab8fe9d1525d">

(above) Shared from my FinalBugTv Site on FB: Works

Results here: https://www.facebook.com/groups/astroid

**Comment by Chacapamac** [comment]
> Shared from Astroid Site on FB: Does not Work

This is weird!!!!

Did you try to clear the Joomla/Astroid cache, it seem to be a possible solution after a fast research —> _Joomla article Open Graph for Facebook work on a website but not on another_

**Comment by Chacapamac** [comment]
Did you check if the Open Graph head Metatags are present for the article you want to share on the Astroid website when viewing the article?

**Comment by paurray** [comment]
@Chacapamac @sonvnn 

a) I tested by logging into my FB account
b) Then going here and entering the links to the article as posted above.
https://developers.facebook.com/tools/debug/
c) I tried again today. Also scrapping again as suggested online. 

No joy. Maybe I am missing something really stupid. Also possible. But as of now. No further ideas or time to trouble shoot again.

New article out later this week. "Deep Dive: Astroid Boilerplate Template 202"

**Reply by sonvnn** [reply]
@paurray I found the problem is Open Graph in template options is Off.
![Screen Shot 2024-03-19 at 10 42 17](https://github.com/templaza/astroid-framework/assets/3204103/31b0e6cf-e010-4254-84ab-d37fb9329c93)

I enabled it on Astroidframe.work. You can try it again.

Thanks,
Sonny

**Reply by sonvnn** [reply, marked as answer]
Just tested with https://astroidframe.work/blog/astroid-hands-on/getting-up-running-the-quickstart-way
And it works very well. 💯

**Comment by paurray** [comment]
@sonvnn That did the trick. All working as expected now 💯 
I am not strictly speaking a developer but...
Would it maybe make sense to have this option switched on by default with every fresh Astroid install?

**Comment by Chacapamac** [comment]
Yeah, I think its good idea to turn it on by default as the possibility to use it is always present in Joomla articles settings.
**Q.**  What will be the reason to turn it off?
